### ISSUE TITLE
Integration not starting on 2024.12.0b0

### ISSUE THREAD

**Issue by Skeletorjus** [nordicopen/easee_hass#525]
### Before creating an issue...

- [ ] There is no open issue describing the same problem
- [ ] I am running latest versions of the integration and of Home Assistant
- [ ] I have checked the operational status of the Easee API

### The problem

The Easee-integration does not start due to missing dependencies.

### Version of Easee integration having the issue?

0.9.65

### Version of Home Assistant Core having the issue?

2024.12.0b0

### Anything in the logs that might be useful for us?

```txt
2024-11-27 22:58:32.319 INFO (SyncWorker_4) [homeassistant.util.package] Attempting install of pyeasee==0.8.9
2024-11-27 22:58:35.075 ERROR (SyncWorker_4) [homeassistant.util.package] Unable to install package pyeasee==0.8.9: × No solution found when resolving dependencies:
      And because pyeasee==0.8.9 depends on pysignalr==1.0.0 and you
      require pyeasee==0.8.9, we can conclude that your requirements are
2024-11-27 22:58:35.076 INFO (SyncWorker_4) [homeassistant.util.package] Attempting install of pyeasee==0.8.9
2024-11-27 22:58:37.350 ERROR (SyncWorker_4) [homeassistant.util.package] Unable to install package pyeasee==0.8.9: × No solution found when resolving dependencies:
      And because pyeasee==0.8.9 depends on pysignalr==1.0.0 and you
      require pyeasee==0.8.9, we can conclude that your requirements are
2024-11-27 22:58:37.350 INFO (SyncWorker_4) [homeassistant.util.package] Attempting install of pyeasee==0.8.9
2024-11-27 22:58:39.998 ERROR (SyncWorker_4) [homeassistant.util.package] Unable to install package pyeasee==0.8.9: × No solution found when resolving dependencies:
      And because pyeasee==0.8.9 depends on pysignalr==1.0.0 and you
      require pyeasee==0.8.9, we can conclude that your requirements are
2024-11-27 22:58:42.791 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'easee': Requirements for easee not found: ['pyeasee==0.8.9'].
```

**Comment by olalid** [comment]
Thanks, we are aware, see https://github.com/nordicopen/pyeasee/issues/118

**Comment by olalid** [comment]
Fixed in 0.9.66
